Tasks/DotNetCoreInstallerV0/externals/install-dotnet.sh

```diff
@@ -654,7 +658,19 @@ install_dotnet() {
     say_verbose "Zip path: $zip_path"
 
     say "Downloading link: $download_link"
-    download "$download_link" $zip_path || download_failed=true
+    download "$download_link" $zip_path || fallback_download_failed=true
+
+    say_verbose "download_failed: $download_failed"
+    say_verbose "valid_legacy_download_link: $valid_legacy_download_link"
+
+    if [ "$fallback_download_failed" = true ]; then
+        say "Cannot download: $download_link"
+        download_link=$fallback_download_link
+        zip_path=$(mktemp $temporary_file_template)
+        say_verbose "Fallback zip path: $zip_path"
+        say "Downloading fallback link: $download_link"
+        download "$download_link" $zip_path || download_failed=true
```

Comment:
This should be fallback_download_failed=true?

Reply:
updated